applications/FluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -33,6 +34,7 @@
 #include "custom_processes/embedded_nodes_initialization_process.h"
 #include "custom_processes/embedded_postprocess_process.h"
 #include "custom_processes/embedded_skin_visualization_process.h"
+#include "custom_processes/integration_point_statistics_process.h"
 #include "custom_processes/move_rotor_process.h"
```

Comment:
Not related to this PR, but maybe it's time to remove this @RiccardoRossi .

Reply:
I am not sure what your refer to...